app/frontend/src/components/base/BaseDialog.vue

```diff
@@ -8,6 +8,10 @@
   >
     <v-card>
       <div class="dialog-body">
+        <div v-if="type === 'X'">
```

Comment:
were already using the name 'type' as the attribute for this component. but i like your suggestion in option 1.. will do

Reply:
did option 1.. 
by default the type attribute is set to OK and shows the OK button.
Mina's concept doesnt have an OK button for this description dialog. so i've changed that to default to false. 
i've checked the other dialogs in the system and theyre all fine because they all specify a 'type' and dont use the default default.